CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+Add apikey to device update
```

Comment:
Is there any associated issue to this?

Reply:
No issue reported

Reply:
I understand at the end we have one, #978, currently added in CNR, so it's fine.

NTC